CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### pg_auto_failover v1.0.5 (Sep 20, 2019) ###
+
+* Fix not being able to create monitor/postgres nodes on mac (#60)
```

Comment:
I think we should also add https://github.com/citusdata/pg_auto_failover/pull/68 (man pages for pg_autoctl)